cmd/licensecheck/main.go

```diff
@@ -10,19 +10,24 @@
 package main
```

Comment:
Please put in a doc comment here, something that explains the output. it's just numbers  and it would be nice to have them explained here.

Similarly a custom usage function would be helpful, perhaps with the same text.

I know I'm asking you to polish a quick hack, but it's a useful tool worth explaining how to use.